README.md

```diff
@@ -43,6 +43,25 @@ docker compose up
 
 - And the Jaeger UI at: <http://localhost:16686>
 
+### Bring your own backend
+
+Likely you want to use the Webstore as a demo application for an observability backend you already have (e.g. an existing instance of jaeger, zipkin or one of the [vendor of your choice](https://opentelemetry.io/vendors/). Adding this backend is easy: Before running docker compose, open the [otelccol-config.yml](./src/otelcollector/otelcol-config.yml) with an editor and add an trace exporter for your backend. For example, if your backend supports otlp extend the `exporters` section by the last 2 lines:
+
+```yaml
+exporters:
```

Comment:
defining exporter alone is not sufficient. this must be added to the pipeline as well.

Reply:
I always forget the pipeline 😂 ... updated.